packages/browser/jest-environment.js

```diff
@@ -13,18 +13,18 @@ class CustomTestEnvironment extends Environment {
      * Solved thanks to https://stackoverflow.com/a/57713960/2133271
      */
     if (typeof this.global.TextEncoder === 'undefined') {
-      const { TextEncoder } = require('util');
+      const { TextEncoder } = await import('util');
       this.global.TextEncoder = TextEncoder;
     }
 
     /**
      * Add support for TextDecoder to JSDOM
      */
     if (typeof this.global.TextDecoder === 'undefined') {
-      const { TextDecoder } = require('util');
+      const { TextEncoder } = await import('util');
```

Comment:
This needs to be Text**Decoder**

```suggestion
      const { TextDecoder } = await import('util');
```